Source/WebCore/dom/Range.h

```diff
@@ -133,6 +131,7 @@ class Range final : public AbstractRange, public CanMakeWeakPtr<Range> {
     RangeBoundaryPoint m_start;
     RangeBoundaryPoint m_end;
     bool m_isAssociatedWithSelection { false };
+    bool isLiveRange() const final { return true; }
```

Comment:
I don't think we should move the declaration of this member function down here.
We usually put all private member variables at the end of class.

Reply:
Got it, I'll move it back to where it was originally.